src/models/currentUser.js

```diff
@@ -28,7 +28,6 @@ export const canEditPost = (currentUser, post) => {
     isAdmin(currentUser)
 }
 
-export const canEditComment = (currentUser, comment) => {
-  // TODO allow moderators to delete comments
-  return same('id', currentUser, comment.user)
-}
+export const canEditComment = (currentUser, comment, community) =>
+  canModerate(currentUser, community) ||
```

Comment:
can you put these on the same line, since they're fewer than 80 chars long? </nitpick>